tests/compiler/test_plugins.py

```diff
@@ -536,32 +553,50 @@ def test_native_types(snippetcompiler: "SnippetCompilationTest", caplog) -> None
             "union_optional_1",
             1.2,
             f"Value 1.2 for argument value of plugin {ns}::union_optional_1 has incompatible type."
-            f" Expected type: Union[int,string]?",
+            f" Expected type: Union[int,string,Type]?",
```

Comment:
What do you think about this? What would be the appropriate type to display here?

Reply:
I'd say `any`. @wouterdb added the `Any` type in his PR. We should probably start using that one instead of `Type` (I thought he had already made the change, but perhaps not).

Reply:
I changed it to the actual inmanta type. It should be `std::Entity` now